addons/pinniped/config-controller/controllers/v3_controller.go

```diff
@@ -75,8 +76,9 @@ func (c *PinnipedV3Controller) Reconcile(ctx context.Context, req ctrl.Request)
 		}
 
 		for i := range secrets.Items {
+			log.Info("Reconciling secret", "secret", secrets.Items[i].Name, "namespace", secrets.Items[i].Namespace)
```

Comment:
Let's make this a V(1) log.  Seems like more of a debug log that we don't need at the top level.

Reply:
so, if something goes wrong, how necessary is this information? it seems necessary but maybe it should be v1?

Reply:
ill cut it.